### DISCUSSION TITLE
Move the one-time verification code higher in email so it shows in mobile notification

### DISCUSSION THREAD

**Discussion by davynpan** [community/community#51038] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be helpful if the code sent to email for verifying a login from an unknown device was in the first few lines of the body so that the code would be displayed on the lock screen notification. Then you could see the code and enter it at a glance.

**Comment by cehfisher** [comment]
Hi @davynpan - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
